panel/widgets/slider.py

```diff
@@ -175,7 +175,7 @@ class DiscreteSlider(CompositeWidget, _SliderBase):
 
     value = param.Parameter()
 
-    value_throttled = param.Parameter(constant=True)
+    value_throttled = param.Parameter()
```

Comment:
Setting these to constant was intentional. Users should generally only set `value` and only use `value_throttled` to watch it. So use the `edit_constant` property to change it internally.